docs/src/references/agent/restart-operation.md

```diff
@@ -8,20 +8,20 @@ sidebar_position: 4
 
 Thin-edge defines a `restart` operation to restart a device, being the main device or a child device.
 
-- A restart is typically triggered by a [mapper](../mappers) on behalf of a cloud operator.
+- A restart is typically triggered by a [mapper](../mappers/index.md) on behalf of a cloud operator.
```

Comment:
Is this mandatory or a best practice?

Reply:
I think it is best practice, but not 100% sure. Though it is inline with all of the references in the docs (all seem to use `/index.md`).